### PR TITLE
Fix missing imports

### PR DESCRIPTION
**Pull request by ulfryk**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -4,8 +4,10 @@
     "baseUrl": "./src",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "importHelpers": true,
```
Comment (varanthir) [inline tsconfig.json]: Is it need necessary? My local env works without it.

File: tsconfig.json [hunk 2]
```diff
@@ -4,8 +4,10 @@
...
     "module": "commonjs",
     "moduleResolution": "node",
+    "noEmitHelpers": true,
```
Comment (varanthir) [inline tsconfig.json]: aa

Comment (ulfryk) [reply tsconfig.json]: There are 2 possible ways to successfully compile and run typescript apps:
- default - TS emits tons of helpers to every compiled file with a decorated class
- `"noEmitHelpers": true` - TS doesn't emit any helpers, but then you need to provide these somehow, and that is where `"importHelpers": true` is useful (it'll work only if `tslib` npm package is provided, see package.json)

Comment (varanthir) [reply tsconfig.json]: Ok, thanks for explanaition. Let's merge it. Info about this should be added to README in next commits.

**Pull request merged**